### DISCUSSION TITLE
Is it possible to disable the indexing of a specific number of rows in search results?

### DISCUSSION THREAD

**Discussion by MaximMonakhov** [davatorium/rofi#2027] [Q&A] [answered]
Hello! Is it possible to disable the indexing of a specific number of rows in search results? I have a combi panel, and the first three lines contain the output from scripts. When I type in the search bar, these lines also appear in the results. 
Alternatively, is it possible to completely disable search for certain mods (or lines)?
![oie_IptGi2fBJCtM](https://github.com/user-attachments/assets/c1b02b8a-5514-4e00-94c3-5fe4e951a98b)
I want to find a telegram in the image, but the script lines prevent me from doing it in 2 characters. The results also include firefox in which there is no 'te' at all.

**Comment by DaveDavenport** [comment, marked as answer]
No, this is not possible (unless you write a c plugin to show the clock/stats, then it might be possible).

**Comment by MaximMonakhov** [comment]
Thanks for the answer, I'll figure it out.
